base/uk.ac.stfc.isis.ibex.ui.dae/src/uk/ac/stfc/isis/ibex/ui/dae/spectraplots/PersistedLocalChannel.java

```diff
@@ -80,4 +84,28 @@ public void onValue(T value) {
     public String getPVAddress() {
     	return address;
     }
+    
+    /**Attempt to write initial value to local PV. If fails (due to an existing race condition), try again until it passes.
+     * @param initialValue - value to be written to PV
+     */
+    private void write_to_pv(T initialValue) {
+    	Runnable task = new Runnable() {
+    		public void run() {
+    			int iterations = 1;
+    			
+    			try {
+    	    		writable.write(initialValue);
+	    		} catch (IOException e) {
+	    			LoggerUtils.logErrorWithStackTrace(LOG, String.format("Couldn't set initial value to '%s' for local PV '%s'. Retrying to re-write...", initialValue, address), e);
+	    			// Restrict writing re-attempts to 30 seconds
+	    			if(iterations < 30) { 
```

Comment:
```suggestion
	    			if (iterations < 30) { 
```

checkstyle complains about missing space otherwise